### ISSUE TITLE
The meaning of input_spec['forecast']

### ISSUE THREAD

**Issue by greatwhiz** [Zenome84/tft_keras#2]
Hi, thanks for your contribution to TFT. I have been actively looking forward to the Keras implementation for a while.

I am not sure about the input_spec['forecast']. Does it mean 'known' columns on the paper?

Thanks for your answers in advance.

**Comment by Zenome84** [comment]
> Hi, thanks for your contribution to TFT. I have been actively looking forward to the Keras implementation for a while.
> 
> I am not sure about the input_spec['forecast']. Does it mean 'known' columns on the paper?
> 
> Thanks for your answers in advance.

Indeed. So this could be, for example, time. However, it is only for the time that has not been observed yet.

I have made updates to the project but have not had a chance to clean it up and upload. I will do so soon. Please check back in a week or so to see how I use it.
